src/Evolution/Executables/Cce/CharacteristicExtractBase.hpp

```diff
@@ -20,8 +20,8 @@
 template <bool EvolveCcm>
 struct CharacteristicExtractDefaults {
   static constexpr bool evolve_ccm = EvolveCcm;
-  using evolved_swsh_tag = Cce::Tags::BondiJ;
-  using evolved_swsh_dt_tag = Cce::Tags::BondiH;
+  using evolved_swsh_tag = tmpl::list<Cce::Tags::BondiJ>;
+  using evolved_swsh_dt_tag = tmpl::list<Cce::Tags::BondiH>;
```

Comment:
Now that these are lists, the name `evolved_swsh_tag` isn't appropriate. Can you make both plural like `evolved_swsh_tags`?

Reply:
Done!